docs/guides/integration-mobile-client-server.md

```diff
@@ -0,0 +1,374 @@
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Link from '@docusaurus/Link';
+import ReduceLatency from '/docs/snippets/_sdk-reduce-latency.mdx';
+import GMAIMAPlugins4GAM from '/docs/snippets/_mobile_docs_plugin_gss.mdx';
+
+# UID2 Client-Server Integration Guide for Mobile
+
+This guide is intended for mobile app publishers who want to integrate with UID2 by generating UID2 tokens server-side via either a public or Private Operator and then passing the tokens and user identities into their mobile apps, which will in turn pass the tokens for bid stream use.  
+
+This is called Client-Server Integration because some integration steps are client-side and some are server-side.
+
+If you want to integrate with UID2 via client-side only changes (that is, all integration changes required are within the mobile apps), refer to the [UID2 Client-Side Integration Guide for Mobile](integration-mobile-client-side.md) instead.
+
+This page provides a high-level overview of integration steps and links to additional documentation.
+
+<!-- It includes the following sections:
+
+- [Overview](#overview)
+- [Complete the UID2 Account Setup](#complete-the-uid2-account-setup)
+- [Client-Server Mobile Integration Data Flow Overview](#client-server-mobile-integration-data-flow-overview)
+- [Implement Server-Side Token Generation](#implement-server-side-token-generation)
+- [Server-Side Token Refresh](#server-side-token-refresh)
+- [Add the UID2 Mobile SDK into Your Mobile App](#add-the-uid2-mobile-sdk-into-your-mobile-app)
+- [Using the UID2 Integration Environment](#using-the-uid2-integration-environment)
+- [Optional: Reduce Latency by Setting the API Base URL for the Production Environment](#optional-reduce-latency-by-setting-the-api-base-url-for-the-production-environment)
+- [Token Storage](#token-storage)
+- [Pass Generated Token for Bid Stream Use](#pass-generated-token-for-bid-stream-use)
+- [Best Practices on When to Pass New UID2 Token to the UID2 SDK](#best-practices-on-when-to-pass-new-uid2-token-to-the-uid2-sdk)
+- [Enable Logging (For Android Only)](#enable-logging-for-android-only)
+- [Enable Automatic Token Refresh in Mobile App/Client Side](#enable-automatic-token-refresh-in-mobile-appclient-side)
+- [Optional: UID2 GMA/IMA Plugin for GAM Secure Signal integration](#optional-uid2-gmaima-plugin-for-gam-secure-signal-integration) -->
+
+## Overview
+
+UID2 provides mobile SDKs for [Android](../sdks/uid2-sdk-ref-android.md) and [iOS](../sdks/uid2-sdk-ref-ios.md). Each SDK has the following features:
+
+- Takes in a UID2 <Link href="../ref-info/glossary-uid#gl-identity">identity</Link> (a UID2 token and associated values) and persists it in local file storage.
+- Automatically refreshes UID2 tokens. (**GWH__SW_01: curious... how often? Do the SDKs take our own advice of refreshing every hour? We don't need to say this, but we could, and even if we don't I'd like to know if we take our own advice.**)
```

Comment:
Thx @sunnywu . Just wondering, since we have a recommended refresh period, whether we follow our own recommendations. Or if the recommendation needs to be updated to include mobile. Prob not.